icu4c/source/data/icu-config.xml

```diff
@@ -292,6 +296,19 @@
 		<include locales="ff"/>
 		<include locales="ff_CM"/>
```

Comment:
Should there be something added to `icu-locale-deprecates.xml` so that the old locale name is aliased to the new one?
(Or do we want to treat both locale names as full locales?)

Reply:
Never mind, I missed the description on the PR where you already addressed this. :)